pyatv/airplay/player.py

```diff
@@ -39,42 +42,43 @@ def play_url(self, url, position=0):
 
         resp = None
         try:
-            resp = yield from self.session.post(
+            session = ClientSession(loop=self.loop)
+            resp = yield from session.post(
                 address, headers=headers, data=body, timeout=TIMEOUT)
-            yield from self._wait_for_media_to_end()
+            yield from self._wait_for_media_to_end(session)
         finally:
             if resp is not None:
                 resp.close()
+            # Apple TV 3 Gen have a bug. It stays in playing state forever
+            # after stream finished. We need close session every time when
+            # video stops to fix it.
+            session.close()
```

Comment:
It's strange that it would behave like that (I haven't seen it on my ATV3). AFAIK the underlying connection should be closed when close() is called. Do you have any reference media that I can try?